packer/HWIMO-BUILD

```diff
@@ -4,7 +4,13 @@
 # input (enviromental variable):
 # * OS_VER        -   ubuntu-14.04 or ubuntu-16.04
 # * BUILD_TYPE    -   vmware or virtualbox
-# * RACKHD_INSTALL_ANSIBLE - which is the ansible yml playbook used. now are rackhd_package or rackhd_local
+# * RACKHD_INSTALL_ANSIBLE - which is the ansible yml playbook used. now are rackhd_ci_builds, rackhd_release or rackhd_local
```

Comment:
Does the comments means the choice for RACKHD_INSTALL_ANSIBLE can be rackhd_ci_builds, rackhd_release or rackhd_local